website/docs/components/stepper/partials/guidelines/guidelines.md

```diff
@@ -9,6 +9,34 @@
 
 - In isolation, consider a standard [icon](/icons/library).
 
+## Steps
+
+### Interactive steps
+
+Interactive steps are part of a sequential flow and can be interacted with as a navigational device.
+
+![](/assets/components/stepper/step-interactive.png)
+
+### Non-interactive steps
+
+Non-interactive steps are a part of a list of steps that can be completed sequentially, out of order, or asynchronously. They cannot be interacted with.
+
+![](/assets/components/stepper/step-noninteractive.png)
+
+## Tasks
+
+### Interactive tasks
+
+Interactive tasks are part of a sequential flow and can be interacted with as a navigational device.
```

Comment:
@jorytindall I pulled this from the old stickersheet, but I'm unsure if it's correct. Do tasks need to be in a sequential flow or is that just for steps? Do you have any ideas on what should be included here if they aren't for sequential flows?

Reply:
I'm trying to remember the reasoning behind this guidance, now that I'm re-reading this it seems a little to restrictive and pointed, especially given that this more or less only a decorative element. The point about the interactive **steps** makes sense, but **tasks** less so. Maybe it would be best to share the content about sequential/out of order/async between both task types, WDYT?

Reply:
Yea, I can use the language @LilithJames-HDS suggested. Thanks!